dev-readme.md

```diff
@@ -9,14 +9,13 @@ Run
 
 In case of errors, try to run it in virtualenv to provide clean Python environment.
 
-### Build & deploy to public repo: temporarily S3
+### Build & deploy public release package 
 
-upload to:
-https://s3.console.aws.amazon.com/s3/buckets/dynatraceawslogforwarder?region=us-east-1&prefix=preview/&showversions=false
+To build a new release, push git tag "release.*" on selected commit. It then triggers entire travis pipeline build and also creates a new release package:
```

Comment:
We could also mention that there's an option to release through GitHub UI but I guess this is good enough for now